dockerfiles/dev.Dockerfile

```
@@ -0,0 +1,22 @@
+FROM golang:1.22
+
+RUN go install github.com/pressly/goose/v3/cmd/goose@latest
+
+WORKDIR /go/src/git.dee.kitchen/buro9/microcosm
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN make build
```

just to check - you've done it this way so that if the go dependencies don't change then it doesn't rebuild that Docker layer? compared to

```suggestion
COPY . .
RUN go mod download
RUN make build
```